server/conf/application.conf

```diff
@@ -52,6 +52,9 @@ whitelabel {
   # The full display name of the civic entity, will use "City of Seattle" if not set.
   civic_entity_full_name = "City of Seattle"
   civic_entity_full_name = ${?WHITELABEL_CIVIC_ENTITY_FULL_NAME}
+
+  favicon_url = "https://civiform.us/favicon.png" # 32x32 pixels
```

Comment:
Nit: Consider moving this up to a comment describing the field as well as the fact that the image should be 32x32 pixels as for the above values. If relevant, I'd also note the supported formats (I can't remember what are valid favicon image formats).

Reply:
done